source/MRViewer/MRViewer.h

```diff
@@ -480,6 +481,7 @@ class MRVIEWER_CLASS Viewer
     MouseUpDownSignal mouseUpSignal; // signal is called on mouse up
     MouseMoveSignal mouseMoveSignal; // signal is called on mouse move, note that input x and y are in screen space
     MouseScrollSignal mouseScrollSignal; // signal is called on mouse is scrolled
+    MouseUpDownSignal mouseClickSignal; // signal is called when mouse button is pressed and immediately released
```

Comment:
please add comment about deferred behaviour of scene controls, and that this signal initialy is emited from mouse controller